src/NzbDrone.Api/Profiles/ProfileSchemaModule.cs

```diff
@@ -27,7 +27,7 @@ private List<ProfileResource> GetAll()
                 .ToList();
 
             var profile = new Profile();
-            profile.Cutoff = Quality.Unknown;
+            profile.Cutoff = Quality.Unknown.Id;
```

Comment:
Ah, because cutoff changed from Quality to int. Does that mean int has dual meaning here: 'quality' or a group id? Wouldn't it be better then to switch to an index in the array?